egs/wsj/s5/utils/data/extract_wav_segments_data_dir.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright    2017  Hossein Hadian
+# Apache 2.0
+
+# This script copies a data directory (which has a 'segments' file), extracting
+# wav segments (according to the 'segments' file)
+# so that the resulting data directory does not have a 'segments' file anymore.
+
+. utils/parse_options.sh
+. ./path.sh
+
+if [ $# != 2 ]; then
+  echo "Usage: $0 <srcdir> <destdir>"
+  echo " This script copies data directory <srcdir> to <destdir> and gets"
```

Comment:
This script will be very slow. Split the data and run the command in parallel.

Reply:
If you do ad --nj and --cmd options to this script, don't set the default of --nj to more than 4, for I/O reasons.